packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -145,6 +146,7 @@ const InternalDashboard: React.FC<InternalDashboardProperties> = ({ onSave, edit
       x: Math.floor(x),
       y: Math.floor(y),
       z: 0,
+      ...(widgetPresets.type === 'table' && { pageSize: DEFAULT_PREFERENCES.pageSize }),
```

Comment:
see comment on the types.ts file, moving the property into the `properties` key of the widget will remove the need for this line entirely.

Reply:
Resolved